### PR TITLE
Add Gherkin to languages and frameworks

### PR DESCRIPTION
**Pull request by TommasoLencioni**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/languages-and-frameworks/gherkin.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Gherkin"
+ring: adopt
+quadrant: "languages-and-frameworks"
+tags: [all]
+---
+
+Gherkin is a domain-specific language designed for describing software behavior without detailing how that behavior is implemented. It serves as a collaboration tool between technical and non-technical team members.
```
Comment (gunzip) [inline radar/languages-and-frameworks/gherkin.md:8]: I'd insert at least link to some helpful resource.

File: radar/languages-and-frameworks/gherkin.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- **Behavior-Driven Development**
+
+- **Structured Format**
+
+- **Tool Agnostic**
+
+### Our use cases:
+
+- [Test automation](https://github.com/pagopa/idpay-functional-testing)
```
Comment (gunzip) [inline radar/languages-and-frameworks/gherkin.md:22]: Is this IDPay specific or it's used also in other products?

File: radar/languages-and-frameworks/gherkin.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- **Human-Readable Syntax**
+
+- **Behavior-Driven Development**
+
+- **Structured Format**
+
+- **Tool Agnostic**
+
+### Our use cases:
```
[the remark this answers is not in the document]
Comment (TommasoLencioni) [reply radar/languages-and-frameworks/gherkin.md:29]: I expanded all the sections, please check them out and let me know, thank you! 😄

**Comment by gunzip** [comment]
The entry is well-composed. However, I have reservations about including it in the 'adopt' quadrant. This classification typically implies widespread use of the technology across different teams, which may not yet be the case yet. Let's see if someone between CODEOWNERs pushes the button :-)

**Comment by lucacavallaro** [comment]
I propose to change the `ring` from `adopt` to `trial`

When Gherkin will be used outside `idpay` we can move it to `adopt`

**Comment by iwoak** [comment]
@lucacavallaro I was thinking the same thing but then I made this query https://github.com/search?q=org%3Apagopa+language%3AGherkin&type=code 
pagoPA, idpay, interop, pn :O

**Comment by lucacavallaro** [comment]
@iwoak ok, do you filtered out the contributions made by consultants? they often bring into our stack their technologies, but then we do not adopt them

**Comment by iwoak** [comment]
@lucacavallaro nope, but I asked to the internal member of those products whether it was our conscious choice

**Comment by ZorbaDimatteo** [comment]
joining the discussion: in pn we use gherkin (proposed by consultant but adopted by team) because it's easier for our QA team to write and discuss test cases with developers, but also to analyze the results of e2e automatic tests more easily

**Comment by gunzip** [comment]
@ZorbaDimatteo may you share a link to PN repo(s) pointing to tests?

**Review by balanza — APPROVED** [review]
I like the entry, thanks @TommasoLencioni 

I propose just a couple of possible, optional refinements, if you find them useful:
1. review the list of repositories that use gherkin
1. differentiate "use cases" (a list of scenario in which we suggest Gherkin to be an proper solution) from "actual usage" (the list of  using repos); this should help readers fit Gherkin in their solutions

**Comment by gunzip** [comment]
I've added the link provided by @iwoak that lists all repos that uses Gherkin. 

I'd vote to merge this one as is, then refine the entry according to:
> differentiate "use cases" (a list of scenario in which we suggest Gherkin to be an proper solution) from "actual usage" (the list of using repos); this should help readers fit Gherkin in their solutions

**Pull request merged**